### ISSUE TITLE
Community Links Not Working

### ISSUE THREAD

**Issue by rosariorobinson** [systers/portal#91]
When I am NOT logged in as Systers Keeper, I click on any of the Community links and error "Server Error (500)". I'm checking with our/ABI IT department, but wanted to log it here as well.

**Comment by ana-balica** [comment]
@rosariorobinson which Portal are we talking about? The one deployed on http://portal.systers.org/ or the development version?

I have tried to access a Community page on the deployed website being logged out and the page gives 200 OK. Does the problem still persist for you?

**Comment by rosariorobinson** [comment]
Yes, I tried both production and development versions I guest.

So here's the scenario.

1. I can click on the links BEFORE I login and all works well.
2. When I login, I get the server 500 error.
3. When I click on any links, I get the server 500 error.
4. I have to click on Systers/home page, then I am actually logged in.
5. When I click on any community link, I get the server 500 error.

I'll add this to GitHub ticket. Thanks.

Rose

**Comment by ana-balica** [comment]
Ok. I think I identified the problem. 
As I see Rose, you have 3 accounts to production Portal. One is Systers Keeper and other 2 registered with other 2 gmail accounts. The user `rosariorobinson` was lacking a matching Systers User profile. Now it should work - http://portal.systers.org/users/rosariorobinson/

The might be several reason: 
* it was created from the admin panel and not registered from the Register page - this problem was fixed in the new portal
* the profile was created, but got deleted at some point in time.

Try if it works now :)
